client/src/app/EmptyTab.less

```diff
@@ -12,48 +12,102 @@
   bottom: 0;
   left: 0;
 
-  .create-buttons {
+  font-size: 14px;
+  font-weight: 400;
+  font-family: 'IBM Plex Sans';
+
+  .flex-spacer {
+    min-width: 10px;
+    max-width: 40px;
+    width: 2%;
+  }
+
+  h2.welcome-header {
+    font-size: 18px;
+    font-weight: 500;
+    margin-bottom: 49px;
+  }
+
+  .welcome-cards {
+    width: 100%;
+    align-items: center;
+    justify-content: center;
+    display: flex;
+    flex-direction: row;
+  }
+
+  .welcome-card {
     display: flex;
     flex-direction: column;
     align-items: center;
     background-color: var(--color-grey-225-10-97);
     color: var(--color-grey-225-10-15);
     border-radius: 10px;
-    width: 320px;
-    min-height: 405px;
-    margin: 50px 20px 20px 20px;
+    width: 280px;
+    min-height: 424px;
+    padding: 15px;
+    padding-top: 20px;
     position: relative;
 
-    .engineIcon {
-      color: var(--color-grey-225-10-35);
-      position: absolute;
-      top: -32px;
+    h3 {
+      font-size: 16px;
+      font-weight: 500;
+      white-space: nowrap;
     }
 
-    .engine {
+    a {
+      color: var(--color-blue-205-100-50);
+      font-weight: 14px;
 
-      display: flex;
-      flex-direction: column;
-      align-items: center;
-      padding: 50px 0 10px 0;
+      &:hover {
+        color: var(--color-blue-205-100-45);
```

Comment:
The links we use in the Modeler don't have a different hover style (darker text color), so we should maintain consistency in these links too.